src/config.c

```diff
@@ -711,13 +710,15 @@ RRStatus ConfigInit(RedisRaftConfig *c, RedisModuleCtx *ctx)
 {
     int ret = 0;
 
-    *c = (RedisRaftConfig) {
-        .str_conf_ref = RedisModule_CreateString(NULL, "", 0)
+    *c = (RedisRaftConfig){
+        .str_conf_ref = RedisModule_CreateString(NULL, "", 0),
     };
 
     if (validateRedisConfig(ctx) != RR_OK) {
         goto err;
     }
+
+    /* clang-format off */
```

Comment:
This is how you disable clang-format for a part: 
```c
/* clang-format off */
code you formatted yourself
/* clang-format on */ 
```

We should avoid using this. This is the only place right now.